code/data_structures/union_find/union_find.js

```diff
@@ -0,0 +1,55 @@
+class UnionFind {
```

Comment:
Please add open genus comment here ! 👍

Reply:
Kindly make a note it is optional and do not make such comments